webrender/src/device/gfx/device.rs

```diff
@@ -1300,16 +1300,8 @@ impl<B: hal::Backend> Device<B> {
     pub fn set_uniforms(
         &mut self,
         _program_id: &ProgramId,
-        projection: &default::Transform3D<f32>,
+        _projection: &default::Transform3D<f32>,
     ) {
```

Comment:
we could do some `assert_eq!(mCurrentProjection, projection)` just to be sure

Reply:
@kvark Added it in 776540a